Jenkinsfile.baguette

```diff
@@ -62,6 +62,7 @@ pipeline {
                             sh 'make -f docker.Makefile save-invocation-image'
                             dir('_build') {
                                 stash name: 'invocation-image', includes: 'invocation-image.tar'
+                                stash name: 'coverage-invocation-image', includes: 'coverage-invocation-image.tar'
```

Comment:
Missed this the first time -- the `post` section of this `stage` should `rmi` the newly tagged name too.

Reply:
Done